keyboards/kbdfans/kbd67/rev1/rev1.h

```diff
@@ -55,3 +55,17 @@
     { K40, K41,   KC_NO, K43, KC_NO, KC_NO, K46, KC_NO, KC_NO, KC_NO, K4A, K4B, K4C,   K4D,   K4E,   K4F }, \
 }
 
+#define LAYOUT_65_iso( \
+    K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C,      K0E, K0F, \
+    K10,      K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D, K1E, K1F, \
+    K20,      K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B, K2C, K2D,      K2F, \
```

Comment:
This layout is not correct. The backslash keycode should go before the Enter, not above it as it is in ANSI: https://github.com/qmk/qmk_firmware/blob/master/layouts/default/65_iso/default_65_iso/keymap.c

Please also add the layout to the info.json.